examples/speech_to_speech/unity/sequence_generator.py

```diff
@@ -0,0 +1,1068 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
I see that most of this code is borrowed from sequence_generator.py Wondering if there is a way to use the existing implementation without copying it over

Reply:
@sravyapopuri388 I copied it because it was difficult to reuse the current sequence generation without significant change.

Reply:
@sravyapopuri388 I inherited SequenceGenerator.